### ISSUE TITLE
Could you please help with investigate error with meson on Ubuntu in TravisCI system

### ISSUE THREAD

**Issue by vit1251** [mesonbuild/meson#3658]
Hello, could you please help me with investigate problem in my Travis CI build in https://travis-ci.org/stephane/libmodbus/jobs/386786823

**Comment by nirbheek** [comment]
You are trying to install meson using Python 2's pip.

**Comment by vit1251** [comment]
I should replace on `pip3 install` ?

**Comment by nirbheek** [comment]
Yes, that should work. However, note that Travis ships with an extremely old version of Ubuntu (14.04), so you may want to use a docker instance in case building your project requires newer versions of libraries,.

**Comment by vit1251** [comment]
Could I ask you please point me in documentation about how to do it over docker?

**Comment by nirbheek** [comment]
See: http://mesonbuild.com/Continuous-Integration.html#continuous-integration
